src/imputation/tmi_imputation.py

```diff
@@ -367,11 +367,12 @@ def apply_tmi(df, target_variables, mean_dict):
     filtered_df = filter_by_column_content(
         df, "status", ["Form sent out", "Check needed"]
     )
-
     # Filter out any cases where 200 or 201 are missing from the imputation class
     # This ensures that means are calculated using only valid imputation classes
     # Since imp_class is string type, any entry containing "nan" is excluded.
     filtered_df = filtered_df[~(filtered_df["imp_class"].str.contains("nan"))]
+    # Only apply TMI to values that are yet to be imputed
+    filtered_df = filtered_df[filtered_df["imp_marker"] == "no_imputation"]
```

Comment:
This can be removed- I've split the df at the very start of TMI so that TMI steps 2 and 4 don't apply to the CF/ MoR rows